plugins/com.google.cloud.tools.eclipse.appengine.validation/plugin.xml

```diff
@@ -54,5 +54,37 @@
       markerType="com.google.cloud.tools.eclipse.appengine.validation.blacklistMarker">
     </markerResolutionGenerator>
   </extension>
+  
+    <extension 
+      name="Web Xml Validator" 
```

Comment:
Should be internationalized into plugin.properties.  Do you think it should mention that it's a Google App Engine Standard validator?